src/common/mfu_flist_walk.c

```diff
@@ -409,7 +418,9 @@ static void walk_readdir_create(CIRCLE_handle* handle)
 
         /* stat top level item */
         struct stat st;
-        int status = mfu_lstat(path, &st);
+        mfu_file_t* mfu_file = *pfile;
+        int status = mfu_file_lstat(path, &st, mfu_file);
+
```

Comment:
```suggestion
        mfu_file_t* mfu_file = *pfile;
        int status = mfu_file_lstat(path, &st, mfu_file);
```

Reply:
Just deleting a line here to keep the code that checks a return code adjacent to where the return code value is created.